sidebars.js

```diff
@@ -572,6 +572,11 @@ const sidebars = {
               label: "Web apps",
               id: "developer-docs/security/web-app-development-security-best-practices",
             },
+            {
+              type: "doc",
+              label: "References",
```

Comment:
```suggestion
              label: "Further reading",
```

Reply:
Using "Important resources" here as this is what I'm now using in security-best-practices-references.mdx as well